mrpt_graphslam_2d/CMakeLists.txt

```diff
@@ -18,3 +18,49 @@ if (BUILD_TESTING)
     ${test_output_directory}/${PROJECT_TEST_NAME})
 endif()
 
+set(PROJECT_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../)
+
+#V-REP Config
+################################################################################
+
+if (EXISTS "${PROJECT_ROOT_DIR}/../V-REP")
```

Comment:
@bergercookie This doesn't look generic enough, hmmm.... :-)

Reply:
You probably don't have to either.. Just use the ENV variable that you have already set...
Use something like $ENV{"VREP_ROOT"}